### ISSUE TITLE
Infobar redesign

### ISSUE THREAD

**Issue by vomikan** [moneymanagerex/moneymanagerex#3224]
I suggest to replace wxStaticText that reproduce selected transaction info or tips to wxWebView.

![image](https://user-images.githubusercontent.com/6836805/114270871-5bccb600-9a17-11eb-8db0-40c36907c5fb.png)

In this case, you can display active links to external resources or internal resources, such as transactions, accounts, and so on, in this dashboard.

**Comment by tactilis** [comment]
I agree that the presentation of the Info area at the bottom of the screen could be improved.

Will need to consider any changes in conjunction with https://github.com/moneymanagerex/moneymanagerex/issues/4106 if that proposal is implemented.
